js/ui/codeView.js

```diff
@@ -453,6 +456,44 @@ var CodingSession = new Lang.Class({
         return this._toolbox;
     },
 
+    setGrabbed: function(grabbed) {
+        if (this._button)
+            this._button.opacity = grabbed ? 0 : 255;
+
+        if (this.toolbox && this._state == STATE_APP)
+            this.toolbox.opacity = grabbed ? 0 : 255;
```

Comment:
**Nitpicky-pick:** I'd create a var with the opacity value just not to repeat those two lines, i.e.
`let opacity = grabbed ? 0 : 255;`

As indicated, this is a nitpick, so feel free to change it only if you think it's worth it.